lib/src/main/java/nl/knaw/dans/lib/dataverse/model/dataset/License.java

```diff
@@ -28,4 +28,8 @@ public class License {
     private String name;
     private URI uri;
     private URI iconUri;
+    private String rightsIdentifier;
+    private String rightsIdentifierScheme;
+    private String schemeUri;
```

Comment:
@aliassheikh can you change the type of schemeUri to URI (compare the uri field on line 29).